src/views/statutory/BoardView.vue

```diff
@@ -260,6 +260,9 @@ export default {
     canEditSelectedBody () {
       return this.can.set_board_comment_and_participant_type.global
         || this.can.set_board_comment_and_participant_type[this.selectedBody]
+    },
+    nonCancelledApplications () {
+      return this.applications.filter(app => !app.cancelled)
```

Comment:
Not sure yet why, but it seems that cancelled applications are not sent to the BoardView. Even if you click the button in the view table.
At first sight it looks fine in the listBoardView function so it's being set somewhere else in statutory. https://github.com/AEGEE/statutory/blob/43ab951729e9d3580ec40f58775c069062c1322f/lib/applications.js#L193-L195